charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -98,23 +98,23 @@ spec:
         readinessProbe:
           httpGet:
             port: 8444
-            path: /healthz
+            path: /healthz/ready
             scheme: HTTPS
           failureThreshold: 1
-          initialDelaySeconds: 20
+          initialDelaySeconds: 40
```

Comment:
I don't think it's necessary to increase this. There's nothing wrong with the readiness probe being hit after 20 (or even just after 5 seconds), as the pod starts out as NotReady and a failed probe won't do anything. 

By increasing this to 40, the controller will always be marked as NotReady for at least the first 40 seconds of its life, even if is actually ready after 20 seconds.

Reply:
yes, agreed